packages/atomic/src/components/commerce/atomic-commerce-recommendation-list/atomic-commerce-recommendation-list.tsx

```diff
@@ -359,20 +343,32 @@ export class AtomicCommerceRecommendationList
   }
 
   private renderAsGrid(product: Product, i: number) {
-    const atomicProductProps = this.getAtomicProductProps(product);
+    const propsForAtomicProduct = this.getAtomicProductProps(product);
+    const {interactiveProduct} = propsForAtomicProduct;
```

Comment:
```suggestion
    const {interactiveProduct, ...propsForAtomicProduct} = this.getAtomicProductProps(product);
```

Reply:
Dismissing since interactiveProduct must be passed as a prop to atomic-product anyways